src/getting-started/use-cases.md

```diff
@@ -0,0 +1,149 @@
+  />
+  <VPCard
+    logo="/assets/icons/proxy.svg"
+    title="SSE Proxy with JWT"
+    desc="A Zilla API gateway setup that proxies SSE traffic with JWT authentication."
+    link="https://github.com/aklivity/zilla-examples/tree/main/sse.proxy.jwt"
+  />
+</div>
+
+### For MQTT Services
+
+<div class="use_cases_cards">
+  <VPCard
+    logo="/assets/icons/data governance.svg"
+    title="MQTT Kafka Broker"
+    desc="A Zilla API gateway setup acting as an MQTT broker with Kafka integration."
+    link="https://github.com/aklivity/zilla-examples/tree/main/mqtt.kafka.broker"
+  />
+  <VPCard
+    logo="/assets/icons/security.svg"
+    title="MQTT Kafka Broker with JWT"
+    desc="A Zilla API gateway setup that provides an MQTT broker with Kafka integration and JWT authentication."
+    link="https://github.com/aklivity/zilla-examples/tree/main/mqtt.kafka.broker.jwt"
+  />
+</div>
+
+### For Kafka Services
+
+<div class="use_cases_cards">
+  <VPCard
+    logo="/assets/icons/connecting kafka.svg"
+    title="Kafka Broker"
+    desc="A Zilla API gateway setup that acts as a Kafka broker for managing message streams."
+    link="https://github.com/aklivity/zilla-examples/tree/main/kafka.broker"
+  />
+  <VPCard
+    logo="/assets/icons/proxy.svg"
+    title="HTTP to Kafka CRUD"
+    desc="A Zilla API gateway setup enabling CRUD operations over HTTP with Kafka integration."
+    link="https://github.com/aklivity/zilla-examples/tree/main/http.kafka.crud"
+  />
+  <VPCard
+    logo="/assets/icons/security.svg"
+    title="Secure Public Access"
+    desc="A secure Public Access Proxy allows authorized Kafka clients to connect to your Amazon MSK cluster or Confluent Cloud via the Internet."
+    link="../solutions/concepts/kafka-proxies/secure-public-access.md"
+  />
+  <VPCard
+    logo="/assets/icons/scalability.svg"
+    title="IoT Ingest and Control"
+    desc="The IoT Ingest and Control Broker lets authorized Kafka clients connect, publish messages, and subscribe to topics via the internet."
+    link="../solutions/concepts/kafka-proxies/iot-ingest-control.md"
+  />
+  <VPCard
+    logo="/assets/icons/use cases.svg"
+    title="Web Streaming"
+    desc="Expose your Kafka clients to the internet via REST and SSE API."
+    link="../solutions/concepts/kafka-proxies/iot-ingest-control.md"
+  />
+</div>
+
+### For gRPC Services
+
+<div class="use_cases_cards">
+  <VPCard
+    logo="/assets/icons/proxy.svg"
+    title="gRPC Proxy"
+    desc="A Zilla API gateway setup that forwards gRPC requests to an upstream server."
+    link="https://github.com/aklivity/zilla-examples/tree/main/grpc.proxy"
+  />
+  <VPCard
+    logo="/assets/icons/connecting kafka.svg"
+    title="gRPC Kafka Echo"
+    desc="A Zilla API gateway setup that echoes messages over Kafka using gRPC."
+    link="https://github.com/aklivity/zilla-examples/tree/main/grpc.kafka.echo"
+  />
+</div>
+
+## AsyncAPI Kafka Gateway
```

Comment:
Add another for `OpenAPI` with these examples:

https://github.com/aklivity/zilla-examples/tree/main/openapi.asyncapi.proxy
https://github.com/aklivity/zilla-examples/tree/main/openapi.proxy